scripts/Level.gd

```diff
@@ -0,0 +1,9 @@
+extends Node
+
+export(String) var level_name
+
+signal start_level
+
+func _ready():
+	if has_node("HUD"):
```

Comment:
Not sure what our the best practices are yet, but I would log a warning here if "HUD" is not present... unless we want it to be really optional.

Reply:
Sure, can do for now.